src/main/scala/org/broadinstitute/dsde/rawls/webservice/AdminApiService.scala

```diff
@@ -57,6 +61,43 @@ trait AdminApiService extends HttpService with PerRequestCreator with UserInfoDi
           WorkspaceService.props(workspaceServiceConstructor, userInfo),
           WorkspaceService.AdminAbortSubmission(workspaceNamespace,workspaceName,submissionId))
       }
+    } ~
+    path("admin" / "groups") { //create group
+      post {
+        entity(as[RawlsGroupRef]) { groupRef =>
+          requestContext => perRequest(requestContext,
+            WorkspaceService.props(workspaceServiceConstructor, userInfo),
+            WorkspaceService.CreateGroup(groupRef))
+        }
+      }
+    } ~
+    path("admin" / "groups" / Segment) { (groupName) => //delete group
+      delete {
+        requestContext => perRequest(requestContext,
+          WorkspaceService.props(workspaceServiceConstructor, userInfo),
+          WorkspaceService.DeleteGroup(groupName))
+      }
+    } ~
+    path("admin" / "groups" / Segment / "members" / Segment) { (groupName, memberEmail) => //add member to group
+      put { //should probably be a POST
```

Comment:
i actually disagree with my own comment here now (I had changed the implementation and forgot to remove the comment.) PUT is also consistent with the admin endpoints that already exist.